infrastructure-playbooks/purge-cluster.yml

```diff
@@ -287,8 +287,6 @@
      state: absent
 
   tasks:
-  - include_vars: roles/ceph-common/defaults/main.yml
-  - include_vars: roles/ceph-osd/defaults/main.yml
```

Comment:
I understand the change for testing, but for a regular use case, wouldn't a user need to have these defined here? Or it should all still work correctly without including these files/vars?